Documentation/Signoffs/Shifter.md

```diff
@@ -0,0 +1,50 @@
+Gear Shifter Subsystem
+--------------------------------------
+### Function of the Subsystem
+The gear shifter subsystem will capture the OEM signal from the gear shifter sensor by splicing into the wiring and storing the signal on a local microcontroller. 
+### Specifications & Constraints
+-	Must use OEM gear shifter sensor
+-	Must not have any exposed or loose wires according to NESC standard
+-	Shall transmit the gear shifter data at a refresh rate no slower than 100 ms
+-	Shall have the same accuracy as the OEM gear shifter sensor
+-	Shall store the signal on a local microcontroller
+
+The refresh rate of less than 100 ms was chosen based off the acceptable latency of modern video games being between 20 ms and 100 ms [1]. Since human reaction time is around 200 ms, this refresh rate would be undetectable by the user of the simulation.
+
+### Wiring Schematic
```

Comment:
I would suggest having isolation so that you don't fry the arudino if the power system of the car does something odd.

Reply:
Also, how do you know that the Arduino is fast enough to detect the falling edge?

Reply:
Also, I don't see much about the tests that you ran to base this off of.

Reply:
What do you mean by having isolation? The Arduino will be fast enough because the human eye was able to see the falling edge on the DMM, and an Arduino is much faster than human reaction time. I guess the refresh rate does not really need to be faster than human detection. In a car you clearly see when the gear changes, and it directly corresponds to when you move the shifter. I will reword some stuff for clarity.

Reply:
Since the wires are being spliced straight off the OEM sensor, wouldn't the Arduino have the same power protection as the OEM sensor due to the ECU's overvoltage protection circuits?

Reply:
Perhaps, but you then need to state what protection is present and then show that it is sufficient to protect your system as well.